docs/api-and-integration/metadata/jira.md

```diff
@@ -1,75 +1,122 @@
 
-The second level of integration is providing links to Jira for the work items that are found. This occurs once the BaseURL is set on the extension.
+* Learn about [Connecting Jira Server and Octopus Deploy](#connecting-jira-server-and-octopus-deploy)
+* Learn about [Connecting Jira Cloud and Octopus Deploy](#connecting-jira-cloud-and-octopus-deploy)
 
-These first two levels of integration work whether your Jira instance is on-premises or SaaS. The third level of integration allows Octopus to update Jira when deployments occur, and **is only available with the SaaS version of Jira**. It becomes enabled once the Connect App is configured (see below) and the Octopus server's URI is set in {{Configuration,Nodes}}.
+---
 
-### Username/Password
+## Connecting Jira Server and Octopus Deploy
 
-If you specify a username and password/apikey then Octopus will retrieve work item descriptions when viewing package details and when creating releases. This works across all versions of Jira.
+This section describes how to configure Octopus Deploy to connect to Jira Server. When configured, this integration always connects to Jira (either from Octopus server itself or via links in the browser). There are no cases where Jira connects to Octopus.
 
-The Jira Connect App password, which is used to push deployment data to Jira, is separate to this due to the nature of the way that specific Jira API works. It is only available in the cloud version of Jira and you must connect to it via a connect app.
+This means you can configure any Octopus instance, whether self-hosted or cloud, to use the Jira integration. The only network connectivity requirements are that your Octopus server and your browser can connect to the Jira Base URL discussed below.
 
-### Release Note Prefix
+If you are using Jira Server, see [Connecting Jira Cloud and Octopus Deploy](#connecting-jira-cloud-and-octopus-deploy).
 
-Once you've specified a username/password, if you also specify a release note prefix the Octopus extension will look through the issue comments for one that starts with that prefix. If it finds one it will use the text following the prefix as the `WorkItemLink.Description`. If you leave this field blank, or a comment starting with the prefix isn't found, the issue's title will be used for the `WorkItemLink.Description`.
+1. Configure the Jira extension in Octopus Deploy.
 
-## Octopus Connect App and the Jira Extension
+    In the Octopus web portal, navigate to **{{Configuration,Settings,Jira Issue Tracker}}** and enter the following values for your Jira instance:
 
-The first step to configure the integration is adding the Octopus Connect App in the Jira marketplace.
+    - **Jira Base URL**. This tells Octopus where your Jira instance is located and enables Octopus to render the links back to Jira issues. i.e., https://your-jira-instance.atlassian.net or https://your-internal-jira-instance/
+    - **Jira Connect App Password**.
```

Comment:
Should we say where in Jira this password can be found?

Reply:
@slewis74 Should this still be here? What would it be for a non-cloud instance? Should we remove it or say ignore it?

Reply:
It shouldn't be here, is only valid for Jira Cloud. And even then, you can't see it anywhere in Jira. It gets generated when you first install the plugin into Jira and then you can't see/edit it after that iirc

Reply:
Removed! :)